src/components/Pages/Event/EventDetailCard/index.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+import moment from 'moment-timezone';
+import { events as i18n, locations as i18n2 } from 'js/i18n/definitions';
+
+import EventLocationDetail from 'components/Pages/Event/EventDetailCard/EventLocationDetail';
+import EventDetailFees from 'components/Pages/Event/EventDetailCard/EventDetailFees';
+
+const EventTime = ({ startTime, endTime }) => {
+  if (startTime) {
+    return (
+      <div className="coa-EventDetailItem__time">
+        {startTime && endTime
+          ? `${moment(startTime, 'HH:mm:ss').format('LT')}—${moment(
+              endTime,
+              'HH:mm:ss',
+            ).format('LT')}`
+          : moment(startTime, 'HH:mm:ss').format('LT')}
+      </div>
+    );
+  }
+
+  return null;
+};
+
+const EventDetailCard = ({
+  date,
+  startTime,
+  endTime,
+  location,
+  eventIsFree,
+  fees,
+  registrationUrl,
+}) => {
+  const intl = useIntl();
+  moment.locale(intl.locale);
+
+  return (
+    <div className="coa-EventPage__EventDetailCard">
+      <div className="coa-EventDetailItem">
+        <i className="material-icons">event</i>
+        <div className="coa-EventDetailItem__content">
+          <div className="coa-EventDetailItem__date">
+            {moment(date, 'YYYY-MM-DD').format('dddd • LL')}
+          </div>
+          <EventTime startTime={startTime} endTime={endTime} />
+        </div>
+      </div>
+      {location && location.locationType === 'city_location' ? (
```

Comment:
we could probably do something with the API on joplin to only return one or the other here, so instead we had:
```
location {
  locationType
  title
  city
  street
}
```

but for now:
```
location {
  locationType
  cityLocation {
    stuff
  }
  remoteLocation {
     stuff
  }
  additionalDetails
}
```
works

Reply:
added a note to the fragment in queries